components/coda/actions/list-docs/list-docs.mjs

```diff
@@ -0,0 +1,121 @@
+import coda from "../../coda.app.mjs";
+
+export default {
+  key: "coda_list-docs",
+  name: "List Docs",
+  description: "Returns a list of Coda docs accessible by the user. These are returned in the same order as on the docs page: reverse chronological by the latest event relevant to the user (last viewed, edited, or shared).",
+  version: "0.0.26",
+  type: "action",
+  props: {
+    coda,
+    isOwner: {
+      propDefinition: [
+        coda,
+        "isOwner",
+      ],
+    },
+    isPublished: {
+      propDefinition: [
+        coda,
+        "isPublished",
+      ],
+    },
+    query: {
+      propDefinition: [
+        coda,
+        "query",
+      ],
+    },
+    sourceDoc: {
+      propDefinition: [
+        coda,
+        "sourceDoc",
+        (c) => c,
+      ],
+      description: "Show only docs copied from the specified doc ID.",
+    },
+    isStarred: {
+      propDefinition: [
+        coda,
+        "isStarred",
+      ],
+    },
+    inGallery: {
+      propDefinition: [
+        coda,
+        "inGallery",
+      ],
+    },
+    workspaceId: {
+      propDefinition: [
+        coda,
+        "workspaceId",
+      ],
+    },
+    folderId: {
+      propDefinition: [
+        coda,
+        "folderId",
+      ],
+      description: "Show only docs belonging to the given folder.",
+    },
+    limit: {
+      propDefinition: [
+        coda,
+        "limit",
+      ],
+    },
+    pageToken: {
+      propDefinition: [
+        coda,
+        "pageToken",
+      ],
+    },
+    paginate: {
+      propDefinition: [
+        coda,
+        "paginate",
+      ],
+    },
+  },
+  async run() {
+    var params = {
+      isOwner: this.isOwner,
+      isPublished: this.isPublished,
+      query: this.query,
+      sourceDoc: this.sourceDoc,
+      isStarred: this.isStarred,
+      inGallery: this.inGallery,
+      workspaceId: this.workspaceId,
+      folderId: this.folderId,
+      limit: this.limit,
+      pageToken: this.pageToken,
+    };
+    Object.keys(params).forEach((key) => (params[key] === null
```

Comment:
Is there a reason you need this? `axios` should strip `null` and `undefined` properties

Reply:
Yes, there is no need for that. Removed this part.
Issue addressed